<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetPasswordDialog</class>
 <widget class="QDialog" name="GetPasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>216</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Key Type:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Password</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Key file</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Key file + Password</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="gb_password">
     <property name="title">
      <string>Enter Password</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="le_password">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="gb_keyfile">
     <property name="title">
      <string>Enter Keyfile</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>File Path:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_keyfile"/>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_password</tabstop>
  <tabstop>le_keyfile</tabstop>
  <tabstop>comboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GetPasswordDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GetPasswordDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>GetPasswordDialog</receiver>
   <slot>_select_keyfile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>466</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>GetPasswordDialog</receiver>
   <slot>_combobox_indexchanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_select_keyfile()</slot>
  <slot>_combobox_indexchanged(int)</slot>
 </slots>
</ui>
